tasks/salesforce.py

```diff
@@ -126,6 +126,12 @@ def _process_metadata(self):
             'false',
         )
 
+        # Set record type visibilities for Case
+        self._set_record_type(
+            'Case.{managed}Incident'.format(**namespace_args),
```

Comment:
if/when we move the record types for case to unmanaged, remember to check this and modify if needed. I can't remember how that bit works, there's managed and namespaced, etc...we may not need the {managed} bit.